src/app/page.tsx

```diff
@@ -1,36 +1,57 @@
-import ClientHomeInterface from '@/app/components/ClientHomeInterface'
-import EditorInterface from '@/app/components/EditorInterface'
+import EditorInterface from './components/EditorInterface'
+import SideBarInterface from './components/SideBarInterface'
 
 export const revalidate = 0
 
-// async function listClients() {
-//   const res = await fetch('https://api-beta.copilot.com/v1/clients?limit=100', {
-//     headers: {
-//       'X-API-KEY': process.env.COPILOT_API_KEY as string,
-//     },
-//   })
+async function listClients() {
+  const res = await fetch('https://api-beta.copilot.com/v1/clients?limit=100', {
+    headers: {
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
+    },
+  })
 
-//   if (!res.ok) {
-//     throw new Error('Something went wrong while fetching client list!')
-//   }
+  if (!res.ok) {
+    throw new Error('Something went wrong while fetching client list!')
+  }
 
-//   return res.json()
-// }
+  const { data } = await res.json()
+  return data
+}
+
+async function getCustomFields() {
+  const res = await fetch('https://api-beta.copilot.com/v1/custom-fields', {
+    headers: {
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
+    },
+  })
+
+  if (!res.ok) {
+    throw new Error('Something went wrong while fetching client list!')
+  }
 
+  const { data } = await res.json()
+  return data
+}
+
+// async function getSettings() {
+//   const { data } = await fetch("http://localhost:3000/api/settings").then(res => res.json())
+//   return data
+// }
```

Comment:
do we need both api and server side fetch for same data? @aatbip

Reply:
I encountered cors error while using the copilot API directly in the server or client component. That's why I created these route handlers and surprisingly cors error doesn't appear here. I will remove this one the cors error will be tackled. Please let me know your thoughts @saugatmaharjan